liquidhaskell-boot/src/Language/Haskell/Liquid/GHC/Misc.hs

```diff
@@ -909,7 +909,7 @@ withWiredIn m = discardConstraints $ do
   -- undef <- lookupUndef
   wiredIns <- mkWiredIns
   -- snd <$> tcValBinds Ghc.NotTopLevel (binds undef wiredIns) (sigs wiredIns) m
-  (_, _, a) <- tcValBinds Ghc.NotTopLevel [] (sigs wiredIns) m
+  (_, a) <- tcValBinds Ghc.NotTopLevel [] (sigs wiredIns) m
```

Comment:
This diff would not be needed if Compat offered a wrapper for tcValBinds that only exposed the last component.

Reply:
TBH this function looks to me, as the kids these days would say, hella sus. There must be a better way to do this instead of pretend-typechecking a bunch of type signatures.